doc/nrf/gs_modifying.rst

```diff
@@ -181,41 +283,37 @@ Configuring build types
 
 .. build_types_overview_start
 
-Build types enable you to use different sets of configuration options for each board.
-You can create several build type :file:`.conf` files per board and select one of them when building the application.
-This means that you do not have to use one :file:`prj.conf` file for your project and modify it each time to fit your needs.
+When the ``CONF_FILE`` variable contains a single file and this file follows the naming pattern :file:`prj_<buildtype>.conf`, then the build type will be inferred to be *<buildtype>*.
+The build type cannot be set explicitly.
+The *<buildtype>* can be any string, but it is common to use ``release`` and ``debug``.
 
-.. build_types_overview_end
+For information about how to set variables, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
 
-.. note::
-    Creating build types and selecting them is optional.
-    This is a feature specific to the :ref:`application development in nRF Connect SDK <app_build_system>`.
+The Partition Manager's :ref:`static configuration <ug_pm_static>` can also be made dependent on the build type.
+When the build type has been inferred, the file :file:`pm_static_<buildtype>.yml` will have precedence over :file:`pm_static.yml`.
 
-.. _gs_modifying_build_types_creating:
+The child image Kconfig configuration can also be made dependent on the build type.
+The child image Kconfig file is named :file:`<child_image>.conf` instead of :file:`prj.conf`, but otherwise follows the same pattern as the parent Kconfig.
```

Comment:
> The child image Kconfig file is named :file:`<child_image>.conf`

This is not necessarily true. The child_image configuration can still have name `prj.conf` if it resides in `child_image/its_name/` directory.

Reply:
My mistake. Good catch.